DAS/README.md

```diff
@@ -2,28 +2,91 @@
 
 *This directory contains scripts to log and test various components of the data data-acquisition-system*
 
-## **Basic Setup and Usage**
+
+
+## Basic Setup and Usage
 These steps assume that you are in the data-acquisition-system root directory.
 1. `cd DAS`
 2. `poetry install`
 3. `poetry shell`
-4. Run any DAS command, eg. `python -m das.mqtt_wireless_logger` (or `python mqtt_wireless_logger.py` as appropriate for your current directory)
+4. Run any DAS command, _e.g._ `python -m das.mqtt_recorder`
```

Comment:
Out of interest, what's the reasoning behind using `python -m das.mqtt_recorder` over `python das/mqtt_recorder.py`?

Reply:
It is because the folder structure is a python module so `-m` is needed to load everything in correctly.

Reply:
Still not completely sure on this, for me everything seems to load properly either way.